UnitsNet/QuantityValue.cs

```diff
@@ -20,31 +21,41 @@ namespace UnitsNet
     ///     From 8 (int, long, double, decimal + each nullable) down to 2 (QuantityValue and QuantityValue?).
     ///     This also adds more numeric types with no extra overhead, such as float, short and byte.
     /// </remarks>
-    public struct QuantityValue
+    [StructLayout(LayoutKind.Explicit)]
+    public readonly struct QuantityValue
```

Comment:
Can we get a short description in summary/remarks of xmldoc to help explain the new multiple numeric types thing? This will not be obvious to most.